examples/eu-ehealth2fhir-vaccination-certificate-signed-bbs.jsonld

```diff
@@ -0,0 +1,45 @@
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://w3c-ccg.github.io/vaccination-vocab/context/v1/EU/unstable.json",
```

Comment:
same comment here, I get worried seeing github files directly embedded in VCs.

Reply:
@OR13 Understood. This is a temporary measure, I guess I am still not fully up to speed with the test rig here, will invest more time to understand how we can test this end-end (basic concern is: can I get the docresolver to work in my dev environment prior to submitting the PR, with a simple set up?). I think we can set this all up for "local resolution" within a docker container that can get deployed for testing purposes. Hopefully this should obviate the need for redirect set up during dev cycles until these enhancements are "stable". We will discuss this with @tplooker to refine.

Would that work?